resilience_stats/views.py

```diff
@@ -303,12 +303,13 @@ def run_outage_sim(run_uuid, with_tech=True, bau=False):
         present_worth_factor = annuity(financial.analysis_years, financial.escalation_pct,
                                        financial.offtaker_discount_pct)
     
-    avoided_outage_costs_us_dollars = round( results['resilience_hours_avg'] * \
+    if 'resilience_hours_avg' in results.keys():
+        avoided_outage_costs_us_dollars = round( results['resilience_hours_avg'] * \
```

Comment:
@t-kwasnik this change makes sense, but I wonder if we should also have `avoided_outage_costs_us_bau_dollars`, which would use the `resilience_hours_avg_bau` value created on line 294?

Reply:
@NLaws I actually had that in there but took it out because we haven't been calculating it already. I just wanted to focus on fixing this bug so the production app will work and then see later if we want to add new fields